components/admin_console/data_grid/data_grid_search.tsx

```diff
@@ -28,10 +28,27 @@ class DataGridSearch extends React.PureComponent<Props> {
         return (
             <div className='DataGrid_search'>
                 <div className='DataGrid_searchBar'>
-                    <SearchIcon
+                    <span
                         className='DataGrid_searchIcon'
                         aria-hidden='true'
-                    />
+                    >
+                        <FormattedMessage
+                            id='generic_icons.search'
+                            defaultMessage='Search Icon'
+                        >
+                            {() => (
```

Comment:
Why is this being done as a function? Not quite sure I understand it here

Reply:
It needed to be like that in order to be wrapped in a formatted message block but I just realized I can actually use the font awesome search icon instead of the importing the svg here and scaling it - pushed a commit fixing this